samples_project/ProjectSettings/TagManager.asset

```diff
@@ -11,7 +11,7 @@ TagManager:
   - 
   - Water
   - UI
-  - 
+  - MenuBG
```

Comment:
Is this tag auto added? Where is it used?

Reply:
Good find! I don't think is is being used. So this can be removed.